specification/web/resource-manager/Microsoft.Web/stable/2016-08-01/WebApps.json

```diff
@@ -15651,12 +15624,7 @@
         "properties": {
           "description": "FunctionEnvelope resource specific properties",
           "properties": {
-            "name": {
-              "description": "Function name.",
-              "type": "string",
-              "readOnly": true
-            },
-            "functionAppId": {
+            "function_app_id": {
```

Comment:
Was this a violation? I thought we preferred the other naming convention.

Reply:
This wasn't a violation. It snuck in due to a merge issue with the generated swaggers. Fixed it.